themes/default/content/blog/using-pulumi-securely/index.md

```diff
@@ -0,0 +1,97 @@
+
+meta_image: meta.png
+
+authors:
+    - tushar-shah
+
+tags:
+    - security
+    - pulumi-enterprise
+
+---
+
+Cloud computing’s greatest strength and weakness is the proliferation of a massive number of services globally. In order to adequately assess and mitigate the inherent risks for your company, your customers, and your employees, cloud architects are typically responsible for a massive surface area of potential endpoints and vectors of attack.
+
+<!--more-->
+
+## Cloud Security Posture
+
+AWS likes to talk about a [Shared Responsibility Model](https://aws.amazon.com/compliance/shared-responsibility-model/), their friendly way of saying, “we’ve got our stuff covered and if you’re compromised, it’s almost certainly your fault.”  They have built up practices and recommendations like the AWS Well Architected Framework; pillars of architecture philosophy that give you your best chances to securely use their resources (following these recommendations is easy with [Crosswalk for AWS]({{< relref "/docs/guides/crosswalk/aws" >}}) from Pulumi).  Increasing high profile attacks against private and open source software supply chains has developers increasingly on high alert and conscious of incorporating more security into their devops.
+
+Risks associated with maintaining a good cloud security posture extend to your infrastructure as code with Pulumi and it’s important to leverage the features and capabilities of the Pulumi platform along with good industry practices to secure your code, configuration and secrets.
+
+## Pulumi’s Hierarchy
+
+Pulumi has three levels to consider access for: users, teams and organizations.
+
+[Users]({{< relref "/docs/intro/pulumi-service/accounts" >}}) are tied to personal identities often with an identity provider and should be used wherever a human is using Pulumi.  They should not be used for any systems, tools or pipeline based provisioning.
+
+[Teams]({{< relref "/docs/intro/pulumi-service/teams" >}}) are groups of users within an organization with specific permissions to stacks.  Consider using these to break up access to environments such as dev, staging and production.  You should look to mirror your code repository or IAM teams/groups as much as possible.
+
+[Organizations]({{< relref "/docs/intro/pulumi-service/organizations" >}}) are a space for you to collaborate on shared projects and stacks.  You may have multiple cloud accounts from a single cloud provider or multiple cloud provider accounts tied to an organization in any combination.  It’s recommended that you segment organizations and cloud accounts to limit access and the “blast radius” of security or financial events that may occur within them.
+
+## Security Best Practices with Pulumi
+
+The following recommendations should help you get started and prepare you to scale your company, application and cloud engineering team long term.
+
+## Manage Federated Identity
```

Comment:
```suggestion
### Manage Federated Identity
```

This feels like a sub-heading of the `Security Best Practices with Pulumi` section